src/components/grids.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react'
+import styled from 'styled-components'
+
+const gapSize = 20
+const numberOfColumns = 12
+
+export const Row = styled.div`
+  box-sizing: border-box;
+  display: flex;
+  flex: 0 1 auto;
+  flex-direction: row;
+  flex-wrap: wrap;
+  margin-right: -${gapSize / 2}px;
+  margin-left: -${gapSize / 2}px;
+  margin-bottom: ${gapSize}px;
+`
+
+export const Col = styled.div`
+  box-sizing: border-box;
+  flex: 0 0 auto;
+  padding-right: ${gapSize / 2}px;
+  padding-left: ${gapSize / 2}px;
+  flex-basis: ${props => (100 / numberOfColumns) * (props.size)}%;
+  max-width: ${props => (100 / numberOfColumns) * (props.size)}%;
```

Comment:
It seems like `flex` is just the shorthand for `flex-grow`, `flex-shrink` and `flex-basis`, so I'm actually :+1: with just using `flex-grow` anywhere. I'll also remove the `max-width`. My only concern now is browser compatibility (and what I mean by browser is, obviously, IE11).